### PR TITLE
add formatter toolbox

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
SQL, HTML, CSS and JS Formatter

### CODE REVIEW

**Comment by drodil** [comment]
Nice addition, thanks for the contribution! Will make it available for public asap.

File: plugins/toolbox/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "postpack": "backstage-cli package postpack"
   },
   "dependencies": {
+    "@types/js-beautify": "^1.13.3",
```
Comment (drodil) [inline plugins/toolbox/package.json:32]: Shouldn't these be in devDependencies instead?

**Review by drodil — CHANGES_REQUESTED** [review]

File: plugins/toolbox/package.json (hunk 1 above)
Comment (dpfaffenbauer) [reply plugins/toolbox/package.json]: yes, obviously. changed it

**Comment by drodil** [comment]
Just pushing new release and it will come 1.3.1. If you do next contributions, please follow the conventional commit messages so that the semantic version works correctly, thanks. Also my own mistake not checking this 😄

**Comment by dpfaffenbauer** [comment]
Sorry. Guess a bot to check that would guide new devs better.

**Comment by drodil** [comment]
That would be great contribution :D

**Pull request merged**